sdrf_pipelines/maxquant/maxquant.py

```diff
@@ -13,15 +13,16 @@
 from datetime import datetime
 import os
 import numpy as np
-
+import pkg_resources

```

Comment:
This means we are depending on setuptools, isn't it? Are we fine using python >=3.7? If so we can use importlib.resources instead.